demos/src/demo-stacked-bar.js

```diff
@@ -35,9 +35,6 @@ function createStackedBarChartWithTooltip(optionalColorSchema) {
             .width(containerWidth)
             .grid('horizontal')
             .isAnimated(true)
-            .stackLabel('stack')
```

Comment:
How does this change the rendering of the chart?

Reply:
It doesn't!